Directory.Packages.props

```
@@ -22,11 +23,13 @@
     <PackageVersion Include="Moq" Version="4.20.72" />
     <PackageVersion Include="MySql.Data" Version="9.1.0" />
     <PackageVersion Include="NCrontab" Version="3.3.3" />
-    <PackageVersion Include="NLog" Version="5.3.4" />
+    <PackageVersion Include="NLog" Version="6.0.0" />
+    <PackageVersion Include="NLog.Extensions.Logging" Version="6.0.0" />
     <PackageVersion Include="NLua" Version="1.7.3" />
     <PackageVersion Include="NetCoreServer" Version="8.0.7" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
     <PackageVersion Include="OSVersionExt.NetStd" Version="3.0.1" />
+    <PackageVersion Include="Pomelo.EntityFrameworkCore.MySql" Version="9.0.0-preview.3.efcore.9.0.0" />
```

I would suggest as much as possible to not create a dependency in a `preview` package

```suggestion
    <PackageVersion Include="Pomelo.EntityFrameworkCore.MySql" Version="9.0.0-preview.3.efcore.9.0.0" />
```

---

I would generally agree, but the preview version is the only one that supports EF Core 9 😅 Given AAEmu is still under development and isn't yet "production grade", it's probably not such a big deal to use a preview package

---

From https://github.com/PomeloFoundation/Pomelo.EntityFrameworkCore.MySql/issues/1841#issuecomment-2932154348:

> The only thing that we have not fully implemented yet for the GA release is complete support of [precompiled queries](https://learn.microsoft.com/en-us/ef/core/what-is-new/ef-core-9.0/whatsnew#aot-and-pre-compiled-queries).
>
> Otherwise, we are production-ready (and have been for a long time).
>
> So don't get scared by the -preview.x suffix. Our codebase is backed by over 30,000 tests (and we don't publish any release, including prereleases, until all existing tests are green).